llvm/test/Transforms/LoopVectorize/X86/interleave_short_tc.ll

```diff
@@ -1,59 +0,0 @@
-; Check that we won't interleave by more than half the "best known" estimated trip count.
-
-; The loop is expected to be vectorized by 4 and interleaving suppresed due to
-; short trip count which is controled by "tiny-trip-count-interleave-threshold".
```

Comment:
Deleted this test since we no longer have the concept of a `tiny-trip-count-interleave-threshold`.